components/homepage/sections/ValuesSection.tsx

```diff
@@ -11,8 +11,22 @@ interface FeatureCardProps {
 const features: FeatureCardProps[] = [
   {
     title: "Permissionless.",
-    description:
-      "Anyone can use thirdweb tools without our permission. The code for our contracts, SDKs, dashboard and UI components are open source and available to everyone.",
+    description: (
+      <>
+        Anyone can use thirdweb tools without our permission. The code for our
+        contracts, SDKs, dashboard and UI components are{" "}
+        <TrackedLink
+          category="values"
+          label="open_source"
+          isExternal
+          color="blue.500"
```

Comment:
can we make these links white? I think the blue looks odd in these contexts, they should still stand out enough?